composer.lock

```diff
@@ -396,6 +399,12 @@
                     "Automattic\\Jetpack\\Autoloader\\": "src"
                 }
             },
+            "scripts": {
```

Comment:
I think this change is not from this PR 🤔

Reply:
We seem to have forgotten to update `composer.lock` when adding those; so they'll have to land in master, one way or another.